src/firewall-applet.in

```diff
@@ -420,6 +420,8 @@ class TrayApplet(QtWidgets.QSystemTrayIcon):
             "normal": QtGui.QIcon.fromTheme(self.icon_name),
             "error": QtGui.QIcon.fromTheme(self.icon_name+"-error"),
             "panic": QtGui.QIcon.fromTheme(self.icon_name+"-panic"),
+            "normal-shields_up": QtGui.QIcon.fromTheme(self.icon_name+"-shields_up"),
```

Comment:
We don't have an appropriate icon by this name to use as a fallback if the current theme doesn't have its own version.

Reply:
If you are ok with the idea, I can probably come up with some useful icons.

Reply:
I think it makes sense to have a new icon for the "shields up" action, but for "shields down" we can probably reuse the "normal" icon. I don't know where the current icons originate. Maybe there is another in the set we can use.